Cargo.toml

```diff
@@ -16,7 +16,10 @@ futures-core = "0.3"
 futures-sink = "0.3"
 tokio = "1"
 tokio-util = { version = "0.7.1", features = ["codec", "io"] }
-tracing = { version = "0.1.36", default-features = false } # Transitive dependency through tokio-util, this is for minvers fix
+# tracing-attributes 0.1.27 (required by tracing 0.1.39) is the first version to depend on proc-macro2 > 1.0.60
+# All older proc-macro2 versions use unknown features and break minvers
+# This is a transitive dependency through tokio-util
+tracing = { version = "0.1.39", default-features = false }
```

Comment:
Can't we instead run `cargo add --no-default-features tracing@0.1.39` in CI? The dependency graph would then remain unmodified for downstream users.

Reply:
I submitted https://github.com/tokio-rs/tokio/pull/6434 to solve this once and for all.

Reply:
Turns out this is in fact due to tower. I think noone actually uses the http-integration code at the moment and it should probably be replaced by an export of `make_key`. Let's drop that single method for now so we can completely get rid of the hyper related dev-dependencies just for that example.